sqe-dto/Catalogue.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace SQE.API.DTO
+{
+    public class CatalogueMatchInputDTO
+    {
+        public enum SideDesignation
+        {
+            recto,
+            verso
+        }
+
+        [JsonConverter(typeof(JsonStringEnumConverter))]
+        public SideDesignation catalogSide { get; set; }
+        public string imagedObjectId { get; set; }
+        public uint manuscriptId { get; set; }
+        public string editionName { get; set; }
+        public string editionVolume { get; set; }
+        public string editionLocation1 { get; set; }
+        public string editionLocation2 { get; set; }
+        [JsonConverter(typeof(JsonStringEnumConverter))]
```

Comment:
I think we need a better Typescript generator. These enums should be converted into union string types in Typescript, and that type should be used on the Typescript side.

Reply:
Agreed.  I'll tackle it this week.